docs/Foundations.md

```diff
@@ -0,0 +1,56 @@
+# Foundations
+
+## Current State
+
+To develop on Xilinx FPGAs, there are a couple of workflows:
+
+1. Legacy HLS -> Vivado workflow
```

Comment:
Maybe providing a section that goes through the useful resources for following the legacy HLS workflow might be handy, in a separate md file as an appendix.